### ISSUE TITLE
Snow Overlay Toggle does not have an accessible name

### ISSUE THREAD

**Issue by Danielle254** [guardiansofthegit-collab/elecretanta#482]
The button needs to have an accessible name. Though the snow overlay effect is not relevant for non-sighted users, there are some portion of screen reader users who have vision. 

Or, an executive decision could be made to hide the button from screen readers entirely.
